content/modules/data-table/versions/1.7.0.um

```diff
@@ -0,0 +1,26 @@
+@version 1.5.0
+  @examples
+    @example
+      @html: @inline 1.5.0/simple-example.html
```

Comment:
Yes, as I didn't want to copy the example again, I just added the existing. I want to retain the original examples and only add a new one